xml/vm_security.xml

```diff
@@ -0,0 +1,216 @@
+[ 11.128307] ccp 0000:42:00.1: SEV API:1.55 build:8
+[ 11.135057] ccp 0000:42:00.1: SEV-SNP API:1.55 build:8</screen>
+<para>The message about the SEV-SNP API version indicates the successful initialization of the AMD Secure Processor. Sometimes it happens that these messages do not appear in the kernel log. In this case the BIOS settings or the IOMMU configuration are often the root-cause.</para>
+</step>
+</procedure>
+</sect1>
+<sect1 xml:id="vm-launch-amd-sv-snp-vm">
+<title>Launching an AMD SEV-SNP virtual machine</title>
+<para>
+You can run AMD SEV-SNP protected virtual machines using the &libvirt; framework once the confidential computing kernel is booted and the AMD Secure Processor is initialized.</para>
+<para>&libvirt; has several ways of setting up new virtual machines, this document uses a prepared disk image and the virt-manager graphical user interface.</para>
+<procedure>
+<step><para>Connect virt-manager to the AMD EPYC host and create a new virtual machine.</para></step>
+<step><para>In the Create a new virtual machine window, select the details:</para>
+<itemizedlist><listitem><para>Select how you want to install the operating system.</para></listitem>
+<listitem><para>Select the ISO or CDROM install media.</para></listitem>
+<listitem><para>Select the memory and CPU settings.</para></listitem>
+<listitem><para>Select the required storage details.</para></listitem>
+</itemizedlist></step>
+<step><para>In the fifth step, verify the details and select <guilabel>Customize configuration before install</guilabel>.</para>
+<figure>
+<title>Create Virtual Machine</title>
+<mediaobject>
+<imageobject role="fo">
+<imagedata fileref="vm_security_create_vm.png" width="75%"/>
+</imageobject>
+<imageobject role="html">
+<imagedata fileref="vm_security_create_vm.png" width="75%"/>
+</imageobject>
+</mediaobject>
+</figure>
+</step>
+<step><para>Click <guilabel>Finish</guilabel>.</para></step>
+<step>
+<para>Select the XML tab in the virtual machine configuration window.</para>
+<para>In the XML tab, you can edit the XML configuration of the virtual machine used by the &libvirt; back-end.</para>
+<figure>
+<title><guimenu>XML</guimenu> view of virtual machine configuration</title>
+<mediaobject>
+<imageobject role="fo">
+<imagedata fileref="vm_security_create_vm_xml.png" width="75%"/>
+</imageobject>
+<imageobject role="html">
+<imagedata fileref="vm_security_create_vm_xml.png" width="75%"/>
+</imageobject>
+</mediaobject>
+</figure>
+</step>
+<step>
+<para>
+To protect the virtual machine with AMD SEV-SNP, set the correct firmware by modifying the <literal>os</literal> section as given below:</para>
+<figure>
+    <title>Set firmware</title>
+    <mediaobject>
+    <imageobject role="fo">
+    <imagedata fileref="vm_security_xml_os.png" width="75%"/>
+    </imageobject>
+    <imageobject role="html">
+    <imagedata fileref="vm_security_xml_os.png" width="75%"/>
+    </imageobject>
```

Comment:
Please add with alt text for the HTML output like this:
```
<textobject role="description">
   <phrase>SHORT_DESCRIPTION</phrase>
</textobject>
```

Reply:
Added as given below:
<figure>
    <title>Set firmware</title>
    <mediaobject>
    <imageobject role="fo">
    <imagedata fileref="vm_security_xml_os.png" width="75%"/>
    </imageobject>
    <textobject role="description"><phrase>Set firmware</phrase>
    </textobject>
    <imageobject role="html">
    <imagedata fileref="vm_security_xml_os.png" width="75%"/>
    </imageobject>
    </mediaobject>
    </figure>